src/navigation/tab-bar-icon.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
```

Comment:
I don't really think you need this code at all? Seems like this is all just so you share  `focusedName && props.focused ? focusedName : normalName` which isn't really worth the abstraction

Reply:
I'm open to alternatives! I couldn't find a way to do this without needing this code.

Reply:
Check latest commit I pushed on this branch